### PR TITLE
ICU-20088 fix broken links (format_parse)

### PR DESCRIPTION
**Pull request by robertgmelo**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-20088
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [x] Documentation is changed or added

### CODE REVIEW

File: docs/userguide/packaging/packaging-icu4j.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+---
+layout: default
+title: Packaging ICU4J
```
Comment (markusicu) [inline docs/userguide/packaging/packaging-icu4j.md:3]: This was deleted on purpose in commit 39db04ff14a9253986541d26e5bea06d589315d3

File: docs/userguide/usefrom/jni.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: Java Native Interface (JNI)
```
Comment (markusicu) [inline docs/userguide/usefrom/jni.md:3]: This was deleted on purpose in commit 3badc757e5938a14409b26a119e50cbf45ebb27f

File: docs/userguide/usefrom/index.md [hunk 3]
```diff
@@ -15,3 +15,4 @@ ICU4C can be used from other programming languages and environments. Please
 refer to the subpages listed below for details.
 
 * [How To Use ICU4C From COBOL](cobol.md)
+* [How To Use ICU4C From Java Native Interface (JNI)](jni.md)
```
Comment (markusicu) [inline docs/userguide/usefrom/index.md:18]: Don't add, see comment on jni.md

File: docs/userguide/usefrom/jni.md (hunk 2 above)
Comment (markusicu) [reply docs/userguide/usefrom/jni.md:3]: Part of PR #1019

File: docs/userguide/packaging/packaging-icu4j.md (hunk 1 above)
Comment (markusicu) [reply docs/userguide/packaging/packaging-icu4j.md:3]: PR #1039

File: docs/userguide/usefrom/jni.md (hunk 2 above)
Comment (markusicu) [reply docs/userguide/usefrom/jni.md:3]: Archived at https://github.com/unicode-org/icu4jni/blob/master/userguide.md

**Comment by robertgmelo** [comment]
It makes sense, Markus. I've dropped them from PR. Also closed ICU-21252 as Invalid.
Thanks in advance.

**Pull request merged**